.gitignore

```diff
@@ -79,5 +79,5 @@ node_modules
 siglens
 
 # multinode testing
-ingest*/
+#ingest*/
```

Comment:
If we don't want this, remove it instead of commenting it out

Reply:
Yes, Will update it